dimension-reduction/README.md

```diff
@@ -0,0 +1,29 @@
+# Dimension Reduction
+
+In this module, we'll demonstrate how to use dimension reduction techniques with
+gene expression data obtained from [refine.bio](https://www.refine.bio/).
+Dimension reduction is useful for examining overall patterns of your data.
+Reducing your data to two dimensions allows you to more easily plot your samples
+so you can observe global and local patterns in your dataset.
+
+## Principal Components Analysis (PCA)
+
+This notebook illustrates how to perform PCA and plot the scores using ggplot2.
```

Comment:
Needs link to rendered HTML notebook https://alexslemonade.github.io/refinebio-examples/dimension-reduction/pca_example.nb.html